homeassistant/components/lametric/entity.py

```diff
@@ -0,0 +1,29 @@
+"""Base entity for the LaMetric integration."""
+from __future__ import annotations
+
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC, format_mac
+from homeassistant.helpers.entity import DeviceInfo, Entity
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import LaMetricDataUpdateCoordinator
+
+
+class LaMetricEntity(CoordinatorEntity[LaMetricDataUpdateCoordinator], Entity):
```

Comment:
The coordinator entity already inherits `Entity`.

Reply:
#77260